src/robusta/cli/main.py

```diff
@@ -87,11 +87,16 @@ def set_pod_configs_for_small_clusters(self):
         )
 
 
-def guess_cluster_name():
+def guess_cluster_name(context):
     with click_spinner.spinner():
         try:
             all_contexts, current_context = config.list_kube_config_contexts()
+            if context is not None:
+                for i in range(len(all_contexts)):
+                    if all_contexts[i].get('name') == context:
+                        return context
             if current_context and current_context.get("name"):
+                typer.echo("default")
```

Comment:
no need to add this log here

Reply:
opps! I was trying to debug so added this that time